js/src/Ice/IncomingAsync.js

```diff
@@ -130,7 +130,7 @@ export class IncomingAsync {
         this._instance.initializationData().logger.warning(s.join(""));
     }
 
-    handleException(ex, amd) {
```

Comment:
Unused parameter

Reply:
I would leave this unrelated cleanup out of this PR.

Reply:
Alright, I removed this from the PR. I'll spin it off into it's own separate commit.